### PR TITLE
Add conda version/downloads, discourse badges to readme

### PR DESCRIPTION
**Pull request by ydcjeff**
## What does this PR do?

[Conda](https://anaconda.org/conda-forge/pytorch-lightning)
[Conda](https://anaconda.org/conda-forge/pytorch-lightning)
[Discourse status](https://forums.pytorchlightning.ai/)

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by rohitgr7** [comment]
maybe we should add discourse badge too now. `[Discourse status](https://forums.pytorchlightning.ai/)`

**Comment by rohitgr7** [comment]
Also the next-release is wrong down there. do we need it??

**Comment by Borda** [comment]
cool, could we also get nb downloads from conda?

File: README.md [hunk 1]
```diff
@@ -20,13 +20,16 @@
 
 [![PyPI Status](https://badge.fury.io/py/pytorch-lightning.svg)](https://badge.fury.io/py/pytorch-lightning)
 [![PyPI Status](https://pepy.tech/badge/pytorch-lightning)](https://pepy.tech/project/pytorch-lightning)
+[![Conda](https://img.shields.io/conda/v/conda-forge/pytorch-lightning?label=conda&color=green)](https://anaconda.org/conda-forge/pytorch-lightning)
+[![Conda](https://img.shields.io/conda/dn/conda-forge/pytorch-lightning?color=blue))](https://anaconda.org/conda-forge/pytorch-lightning)
```
Comment (Borda) [inline README.md:24]:
```suggestion
[![Conda](https://img.shields.io/conda/dn/conda-forge/pytorch-lightning?color=blue)](https://anaconda.org/conda-forge/pytorch-lightning)
```

**Review by Borda — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Comment by Borda** [comment]
maybe we shall write also what type of download to the badge as not it is hard to separate what is pypi and conda...

**Comment by ydcjeff** [comment]
@Borda can I do that along in this [PR](https://github.com/PyTorchLightning/pytorch-lightning/pull/3328)?

Since the change is not big...

**Pull request merged**